plugins/modules/instance_info.ps1

```diff
@@ -0,0 +1,47 @@
+#!powershell
+# -*- coding: utf-8 -*-
+
+# (c) 2022, John McCall (@lowlydba)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+#AnsibleRequires -PowerShell ansible_collections.lowlydba.sqlserver.plugins.module_utils._SqlServerUtils
+#Requires -Modules @{ ModuleName="dbatools"; ModuleVersion="1.1.83" }
+
+$ErrorActionPreference = "Stop"
+
+# Get Csharp utility module
+$spec = @{
+    supports_check_mode = $true
+    options = @{}
+}
+
+$module = [Ansible.Basic.AnsibleModule]::Create($args, $spec, @(Get-LowlyDbaSqlServerAuthSpec))
+$sqlInstance, $sqlCredential = Get-SqlCredential -Module $module
+$module.Result.changed = $false
+
+# Fetch instance information
+try {
+    $getSplat = @{
+        SqlInstance = $sqlInstance
+        SqlCredential = $sqlCredential
+        EnableException = $true
+    }
+    $output = Get-DbaConnection @getSplat | Select-Object -Property "ComputerName", "SqlInstance", "InstanceName" -First 1
+
+    if ($null -ne $output) {
+        # Add additional fields
+        $extraProperties = @("BuildNumber", "Language", "VersionMajor", "VersionMinor", "VersionString", "Collation", "ProductLevel", "IsClustered", "LoginMode")
```

Comment:
re: the sanity long line failure, one of the things I like anyway about PowerShell's multi-line array and hashtable syntax is that the commas become optional that way:
```powershell
$a = @(
    "item1"
    "item2"
)

$h = @{
    KeyA = "one"
    KeyB = "two"
}
```
(I knew about the hashtable one for a while, took me forever to realize it worked for arrays too 🤦‍♂️)

Reply:
TIL that for arrays!